README.rst

```diff
@@ -49,13 +49,13 @@ Track configuration
 
 The configuration for a track is a list of triggers, each of which is a dictionary containing the following keys:
 
-- ``start`` is the time of the trigger, in seconds, relative to the game start time.
-- ``group`` (optional) is the exclusivity group to assign this trigger to; only one sound from a given exclusivity group can be playing at a time.
+- ``start`` is the time of the trigger, in seconds, relative to the game start time. Note: This value can be negative to represent actions before the start of the match. The limit is the pre-match time defined in the compstate schedule.
```

Comment:
Either in this PR or separately I think it would be worth documenting why this works. Probably we need a big docstring on either the `Scheduler` or `Mixtape` classes (or both) outlining their relationship to the timings.
I suspect some docstrings on the key methods/functions would also be useful.
I'd be happy to add these separately if you like (which might be useful to help me validate my understanding actually).